pkgs/applications/networking/instant-messengers/discord/linux.nix

```diff
@@ -211,7 +209,9 @@ stdenv.mkDerivation rec {
 
   desktopItem = makeDesktopItem {
     name = pname;
-    exec = binaryName;
+    exec =
```

Comment:
I don't think it should be done in here.

I may be wrong but if you really want to do something similar, you should wrap the binary using `makeBinaryWrapper` and add the option in there.

Reply:
you're right, i moved it from `makeDesktopItem` to `wrapProgramShell`. i hope this is better :)